<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Notify</class>
 <widget class="QWidget" name="Notify">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tray</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="grpEnable">
     <property name="title">
      <string>Enable system tray notifications</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>79</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="chkNotifyFile">
          <property name="text">
           <string>Notify when file is complete</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkNotifyBatch">
          <property name="text">
           <string>Notify when file group is complete</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkNotifyRepair">
          <property name="text">
           <string>Notify on archive repair</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkNotifyExtract">
          <property name="text">
           <string>Notify on archive extract</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkNotifyError">
          <property name="text">
           <string>Notify on errors</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkNotifyWarning">
          <property name="text">
           <string>Notify on warnings</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>grpEnable</tabstop>
  <tabstop>chkNotifyFile</tabstop>
  <tabstop>chkNotifyBatch</tabstop>
  <tabstop>chkNotifyRepair</tabstop>
  <tabstop>chkNotifyExtract</tabstop>
  <tabstop>chkNotifyError</tabstop>
  <tabstop>chkNotifyWarning</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
